run-tests-with-build-optawebs-parent/optawebs-project-sources-test/common.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+# Runs a cypress tests from a container
+# param1: test application URL
+# param2: directory containing a 'cypress' subdirectory with cypress tests
+# param3: cypress image version
+function run_cypress() {
+  local _application_url=$1
+  local _frontend_directory=$2
+  local _cypress_image_version=$3
+
+  podman run \
```

Comment:
just checking - running on podman only?

Reply:
Shell I extend it to be able to use docker as well?

Reply:
Extended